### DISCUSSION TITLE
Error while installing TaillWind for a NextJS project

### DISCUSSION THREAD

**Discussion by kovymun** [tailwindlabs/tailwindcss#8592] [Help]
Hello all,
I am having this issue when installing Tailwind for a NextJS project. I have followed all the steps accordingly as per the instructions on the website, but I am still having this error. I would appreciate your assistance on this

![Image_TW](https://user-images.githubusercontent.com/78513588/173164191-7fabfd8a-1c44-41fa-bbc2-87c33fd645e1.PNG)

**Comment by rohmann** [comment]
Hi @Kovy424 ,

It's hard to know exactly without more info, but I've found errors like this are often related to inconsistencies with your local npm packages. First it'd be good to make sure npm is up to date. Then it might help to do the following

* Delete your local `node_modules` in your project and run `npm install` again.
* If that doesn't work, try also deleting `package-lock.json` then run `npm install`.
* Finally, you could try running `npm cache clean --force` to clean your npm cache, then run `npm install`.

The idea here is to clear any dependency conflicts so all the different packages in your project can jive together.

Hope this helps! If not, posting some more specific log information including your node and npm versions as well as the contents of your `package.json` file would be helpful to get an idea of what's going on.
